src/builders.ts

```diff
@@ -22,3 +49,24 @@ export function buildPattern(
 ): string {
   return encodeSequence(elements).pattern;
 }
+
+function encodeFlags(flags: RegexFlags): string {
```

Comment:
Would it make sense to move `encodeFlags` to `encoder` folder to keep encoders together?

Reply:
`encodeFlags` is only used in `buildRegex', so I would keep it there for now.